lib/index.ts

```diff
@@ -0,0 +1,26 @@
+import { clientInfo } from './environment'
+
+declare var PercyAgent: any;
+
+/**
+ * A function to take a Percy snapshot from a NightmareJS test. To use in your tests:
+ *   const { percySnapshot } = require('@percy/nightmare')
+ *   [...]
+ *   nightmare
+ *     .goto(<yourtesturl>)
+ *     .use(percySnapshot(<your snapshot name>), <maybe options>))
+ *
+ * @param name Name of the snapshot that we're taking.
+ * @param options Additional options, e.g. '{widths: [300, 600, 1000]}'. Optional.
+ */
+export function percySnapshot(name: string, options: any = {}) {
+  return function (nightmare: any) {
+    nightmare
+      .inject('js', 'node_modules/@percy/nightmare/node_modules/@percy/agent/dist/public/percy-agent.js')
```

Comment:
@djones This is a piece that I feel a little uncomfortable: hardcoding the path to the `percy-agent.js`. I did add `@percy/agent` as a dependency to this package, to make sure that it is always available, but I'm wondering if there is a more robust way of getting the path to this file at runtime.

Reply:
yeah I had a similar thought... that file will definitely continue to live there for quite some time but I was wondering if there was any way to get a path to @percy/agent more directly... maybe a bit of research required to see what's possible

Reply:
Updated the code to compute more robustly the path to `percy-agent.js`, with a fallback to this hardcoded version.